web3/eth.py

```diff
@@ -126,10 +125,46 @@ def default_block(self) -> BlockIdentifier:
     def default_block(self, value: BlockIdentifier) -> None:
         self._default_block = value
 
+    @property
+    def defaultBlock(self) -> BlockIdentifier:
+        warnings.warn(
+            'defaultBlock is deprecated in favor of default_block',
+            category=DeprecationWarning,
+        )
+        return self._default_block
+
+    @defaultBlock.setter
+    def defaultBlock(self, value: BlockIdentifier) -> None:
+        warnings.warn(
+            'defaultBlock is deprecated in favor of default_block',
+            category=DeprecationWarning,
+        )
+        self._default_block = value
+
     @property
     def default_account(self) -> Union[ChecksumAddress, Empty]:
         return self._default_account
 
+    @default_account.setter
+    def default_account(self, account: Union[ChecksumAddress, Empty]) -> None:
+        self._default_account = account
+
+    @property
+    def defaultAccount(self) -> Union[ChecksumAddress, Empty]:
```

Comment:
It doesn't really hurt, but I don't think we need to make these older-style camel case methods available on the async_w3 instance 🤷‍♀️

Reply:
I think I'll leave this one as is, just so that the old and new style live next to each other. Unless it's ok to just remove them completely? Something breaking to add to v6?